### PR TITLE
use faster default JS compressor if not compressing offline

### PR DESCRIPTION
**Pull request by biyeun**
@benrudolph

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -1614,7 +1614,8 @@
 }
 
 COMPRESS_CSS_HASHING_METHOD = 'content'
-COMPRESS_JS_COMPRESSOR = 'corehq.apps.style.uglify.JsUglifySourcemapCompressor'
+COMPRESS_JS_COMPRESSOR = 'compressor.js.JsCompressor' if not COMPRESS_OFFLINE else 'corehq.apps.style.uglify.JsUglifySourcemapCompressor'
```
Comment (dimagimon) [inline settings.py]: E501 line too long (137 > 115 characters)

**Comment by benrudolph** [comment]
test failure looks real

**Comment by biyeun** [comment]
@benrudolph

**Comment by benrudolph** [comment]
sweet

**Pull request merged**